ansible/roles/local_build/tasks/main.yml

```diff
@@ -0,0 +1,13 @@
+---
+
+- name: Set path variables
+  set_fact:
+    root_path: "{{ playbook_dir | dirname }}"
+    release_tree_path: "{{ playbook_dir | dirname }}/releases"
+    release_path: "{{ playbook_dir | dirname }}/releases/{{ deployment_environment_id }}"
+
+- import_tasks: go.yml       # check go version
+- import_tasks: node.yml     # install nvm to ensure our node version is recent enough
```

Comment:
so here, we should change this and any other lines where we have a role for installing the stuff under ansible/roles.